lib/monitoring/aws-apigateway/ApiGatewayMetricFactory.ts

```diff
@@ -122,33 +125,33 @@ export class ApiGatewayMetricFactory {
     );
   }
 
+  /**
+   * @deprecated please use metricLatencyInMillis instead
```

Comment:
nitpick: In general we avoid the use of words like `please` and be direct with the ask.

```
@deprecated use metricLatencyInMillis instead
```

Reply:
Makes sense. Will update.